Dockerfile

```diff
@@ -24,6 +24,9 @@ RUN yum update -y \
 RUN pip install -U pip wheel
 
 COPY requirements /pip/requirements/
+
+RUN sed -i 's/simplejson==2.3.2/simplejson>=2.3.2/' /pip/requirements/compiled.txt
```

Comment:
this looks pretty fragile. Can we just bump the version in `compiled.txt` ? I think that would be fine for a couple reasons:
- nothing is actually deployed to prod from `compiled.txt` (it's all done with rpms)
- the simplejson interface is pretty much frozen so I don't think an upgrade will break any code

I'll make one request here though. Instead of `>=2.3.2` can you just pin it with `==` to the version that works?